src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int leftLeg;
+    private int rightLeg;
+
+    public RightTriangle(String color, int leftLeg, int rightLeg) {
+        super(color);
+        this.leftLeg = leftLeg;
+        this.rightLeg = rightLeg;
+    }
+
+    @Override
+    public double area() {
+        return (leftLeg * rightLeg) / 2;
+    }
+
+    @Override
+    public void draw() {

```

Comment:
The `draw()` method here is not conceptually close to the `area()` method. It's better to separate these methods into different interfaces. This is a code quality issue.